### ISSUE TITLE
R6.1: Adapter DRY Phase 2 — forward/output, get_config, comma naming

### ISSUE THREAD

**Issue by petteriTeikari** [minivess-mlops/minivess-mlops#52]
## Context

R5 extracted checkpoint/save/export methods to `ModelAdapter` base (R5.1+R5.2), eliminating ~350 LOC. Three more adapter boilerplate patterns remain.

Ref: `docs/planning/code-review-report-v0-2-2nd-pass.md` — R5.3, R5.5, R5.28

### R5.3: Add `_build_output()` helper to ModelAdapter base
All 7 adapters have nearly identical forward/softmax patterns:
```python
logits = self.net(images)
prediction = torch.softmax(logits, dim=1)
return SegmentationOutput(prediction=prediction, logits=logits, metadata={...})
```
**Files:** All adapter `.py` files + `adapters/base.py`
**Est. tests:** ~2

### R5.5: Add `_build_config()` helper to ModelAdapter base
Each adapter constructs `AdapterConfigInfo` with the same common fields. Only `extras` differs.
**Files:** 6 adapter files + `adapters/base.py`
**Est. tests:** ~2

### R5.28: Standardize comma.py parameter names
`adapters/comma.py` uses abbreviated `in_ch`, `out_ch`, `x` while all other adapters use `in_channels`, `out_channels`, `images`.
**Files:** `adapters/comma.py` + tests
**Est. tests:** ~2

## Acceptance Criteria
- [ ] `_build_output()` in base class, subclasses call it
- [ ] `_build_config()` in base class, subclasses pass only `extras`
- [ ] `comma.py` uses full parameter names matching other adapters
- [ ] All 868+ existing tests still pass
- [ ] ~6 new tests added
- [ ] TDD workflow (RED → GREEN → VERIFY)

## Estimated LOC reduction: ~140 lines

**Comment by petteriTeikari** [comment]
Implemented in R6 remediation sprints. See commits referencing this issue.
